### PR TITLE
Changes : added thank you in punjabi, added type attribute in script, removed empty lines

### PR DESCRIPTION
**Pull request by YashKumarVerma**
- Removed some empty languages.
- Added thank you for contributing in punjabi
- Added `type="text/javascript"` in script tags.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -942,17 +937,16 @@ <h6 class="title">FOSSASIA Links</h6>
...
-        <script src="js/bootstrap.min.js"></script>
-        <script src="js/flexslider.min.js"></script>
-        <script src="js/lightbox.min.js"></script>
-        <script src="js/masonry.min.js"></script>
-        <script src="js/flickr.js"></script>
-        <script src="js/twitterfetcher.min.js"></script>
-        <script src="js/spectragram.min.js"></script>
-        <script src="js/smooth-scroll.min.js"></script>
-        <script src="js/parallax.js"></script>
-        <script src="js/scripts.js"></script>
```
Comment (0x48piraj) [inline index.html:956]: Why done this?

Comment (YashKumarVerma) [reply index.html:956]: i read somewhere that this specifies the MIME type of the linked file.

https://www.tutorialspoint.com/html/html_script_tag.htm

Comment (0x48piraj) [reply index.html:956]: No, I meant you deleted the same lines and re-added it.
And, this dismantled some line spacing. Fix them :+1:

Comment (YashKumarVerma) [reply index.html:956]: oh sure. i'll be closing this pr and making a new one then 👍

Comment (0x48piraj) [reply index.html:956]: No. Just make a commit. And then squash commits :wink:

Comment (YashKumarVerma) [reply index.html:956]: i face a lot of issues in squashing. 
so, i'm closing this

**Pull request closed without merging**